lua/entities/gmod_wire_expression2/core/hologram.lua

```diff
@@ -1276,6 +1278,163 @@ end
 
 -- -----------------------------------------------------------------------------
 
+local function SetHoloAnim(Holo, Animation, Frame, Rate)
+	if (Holo and Animation and Frame and Rate) then
+		if not Holo.ent.Animated then
+			-- This must be run once on entities that will be animated
+			Holo.ent.Animated = true
+			Holo.ent.AutomaticFrameAdvance = true
+
+			local OldThink = Holo.ent.Think
+			function Holo.ent:Think()
+				OldThink(self)
+				self:NextThink(CurTime())
+				return true
+			end
```

Comment:
Should add some logic for animations to the hologram ENT:Think instead of doing this hackery